sotodlib/preprocess/processes.py

```diff
@@ -1450,7 +1450,7 @@ class SplitFlags(_Preprocess):
     name = "split_flags"
 
     def calc_and_save(self, aman, proc_aman):
-        split_flg_aman = obs_ops.flags.get_split_flags(aman, proc_aman, split_cfg=self.calc_cfgs)
+        split_flg_aman = obs_ops.splits.get_split_flags(aman, proc_aman, split_cfg=self.calc_cfgs)
```

Comment:
Though its not included in the changes here, there is a formatting issue for the docstring in `obs_ops.splits`. This might be a good PR to fix that on.  See https://github.com/simonsobs/sotodlib/blob/686e4ab01415deccfbe0e25fb45d331232c688ba/sotodlib/obs_ops/splits.py#L67

Reply:
ok, I'll add this